process/templates/PROJECT_NAME_graduation_stage.md

```diff
@@ -1,7 +1,10 @@
 ## Project graduation application
 
+### Project has met all Incubating requirements
+  * "link to Incubating PR"
+
 ### List of project maintainers
-The project must have maintainers with a minimum of three different organizational affiliations.
+The project must have maintainers with a minimum of five different contributors from three different organizational affiliations.
```

Comment:
I assume we want to update the requirements in the [project lifecycle doc](https://github.com/ossf/tac/blob/main/process/project-lifecycle.md#graduated-project-entry-requirements-and-considerations) to be this specific as well?